index.html

```diff
@@ -1820,11 +1820,12 @@ <h2>Algorithms</h2>
 
       <p>
 The algorithms defined below operate on documents represented as <dfn
-data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows
-[[JSON-LD-API]] in representing a JSON object as an [[Infra]] [=map=]. An
-<dfn class="export">unsecured data document</dfn> is a [=map=] with no
-`proof` [=map/key=]. A <dfn class="export">secured data document</dfn> is a
-[=map=] with a `proof` [=map/key=].
+data-cite="RFC8259#section-4">JSON objects</dfn>. This specification follows the
+[[[JSON-LD11-API]]] specification in representing a JSON object as an
+[[[Infra]]] [=map=]. An <dfn class="export">unsecured data document</dfn> is a
+[=map=] that has not yet had the desired proof added to it, but it MAY contain a
+proof value added to it by a previous process. A <dfn class="export">secured
+data document</dfn> is a [=map=] that has had the desired proof(s) added to it.
```

Comment:
Alternative "target":

```suggestion
[=map=] that has not yet had the target proof added to it, but it MAY contain a
proof value added to it by a previous process. A <dfn class="export">secured
data document</dfn> is a [=map=] that has had the target proof(s) added to it.
```

Reply:
To me, "target" implies "thing to which other things are being applied" rather than "thing being applied to another thing". So, I think, not `target`.

Reply:
I've updated the spec to speak to "unsecured data documents", "input documents", and "secured data documents". The new formulation feels like it addresses various concerns raised throughout this PR.